docs-source/content/faq/fan.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Disabling Fast Application Notifications"
+date:
+draft: false
+---
+
+To support Fast Application Notifications (FAN), Oracle databases configure GRID.
```

Comment:
is GRID an acronym, we should expand it first time

Reply:
It is not an acronym the product is actually called "Oracle Grid Infrastructure".

Reply:
I'll write that out.